templates/terraform/command-center-project/service-accounts.tf

```diff
@@ -0,0 +1,11 @@
+module command_center_service_account {
+  source = "/templates/google-sa"
+  providers = {
+    google.target = google.target
+  }
+
+  account_id = var.service_account_id
+  display_name = "Service account to run GKE system pods"
+  vault_path = "${var.vault_prefix}/gcs/sa-key"
+  roles = ["logging.logWriter", "monitoring.metricWriter", "monitoring.viewer"]
```

Comment:
I think as-written the `google-sa` module assumes these role strings will include the `roles/` prefix. For example: https://github.com/broadinstitute/monster-deploy/blob/master/templates/terraform/command-center-project/cloudsql.tf#L56

That said, I like the idea of moving that hard-coded prefix into the module so call-sites can look like this. Could you update the module accordingly?

Reply:
Yeah! That sounds like a good idea